safeeyes/model.py

```diff
@@ -159,38 +164,55 @@ def next(self):
         return break_obj
 
     def reset(self):
-        self.__short_pointer = self.__short_header
-        self.__long_pointer = self.__long_header
-        self.__first_break = None
-        self.__current_break = None
-
-        # Reset all break time
-        short_pointer = self.__short_pointer
-        long_pointer = self.__long_pointer
-        short_pointer.time = self.__short_break_time
-        long_pointer.time = self.__long_break_time
-        short_pointer = short_pointer.next
-        long_pointer = long_pointer.next
+        for break_object in self.__short_queue:
+            break_object.time = self.__short_break_time
+        
+        for break_object in self.__long_queue:
+            break_object.time = self.__long_break_time
```

Comment:
Looks like we need to call `__build_queue` to create new queues with random order right? Otherwise, we need to restart Safe Eyes to randomize the breaks.

Everything else looks good to me. I'm terribly sorry for the delay. I got an extremely busy week and couldn't spend time on my FOSS projects.

Reply:
Yes, `__build_queue` creates the queue. It is called inside `BreakQueue __init__(...)` every time the settings are changed by `SafeEyesCore initialize(...)` method in `safeeyes/core.py:78` so there is no need to restart.
You can check it by adding a print statement into `BreakQueue __init__(...)` and then swithcing random order on and off. It will rebuild the queue every time.

Reply:
Yup. I am asking about this use case: Some users never restart their computer and they may not even change settings frequently. So for them, once Safe Eyes is started, order of the breaks will remain the same. For those who restart the computer and/or change settings, the current code will change the order.

From an end user's perspective, the `random breaks` option should change the order every time. So I think we need to randomize the short breaks queue once all short breaks are taken and also randomize the long breaks queue once all long breaks are taken.

The reset method may not be the place to make this change. Maybe in the next break method or next short/long break methods.

Reply:
This case is handled by another module already. `SafeEyesCore` class re initializes the queue every time the settings are changed. I just tesed it, it does switch on the fly without restarting.

Reply:
Got it